pkg/api/types.go

```diff
@@ -3808,8 +3808,8 @@ type RangeAllocation struct {
 }
 
 const (
-	// "default-scheduler" is the name of default scheduler.
-	DefaultSchedulerName = "default-scheduler"
+	// "kube-scheduler" is the name of default scheduler.
+	DefaultSchedulerName = "kube-scheduler"
```

Comment:
are there existing scheduler annotations or fields referencing "default-scheduler" this will break?